net-libs/libtorrent/libtorrent-0.13.6.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils libtool toolchain-funcs
+
+DESCRIPTION="BitTorrent library written in C++ for *nix"
+HOMEPAGE="https://rakshasa.github.io/rtorrent/"
+SRC_URI="http://rtorrent.net/downloads/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x64-solaris"
+IUSE="debug ipv6 ssl"
+
+RDEPEND="
+	>=dev-libs/libsigc++-2.2.2:2
+	ssl? ( dev-libs/openssl:0= )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+src_prepare() {
+#	epatch "${FILESDIR}"/download_constructor.diff
+	elibtoolize
+}
+
+src_configure() {
+	# the configure check for posix_fallocate is wrong.
+	# reported upstream as Ticket 2416.
+	local myconf
+	echo "int main(){return posix_fallocate();}" > "${T}"/posix_fallocate.c
```

Comment:
It looks to me as though this whole section can be removed instead. Again it's not possible to reference the upstream bug with the bug tracker being gone but it is correctly detecting the presence of `posix_fallocate()`.

I'll set up a chroot and try to confirm it will work without `posix_fallocate()` available as well, then we may be able to simplify further here.